images/php-fpm/Dockerfile

```diff
@@ -10,12 +10,12 @@ ENV COMPOSER_HOME=
 ENV MAILHOG_HOST    mailhog
 ENV MAILHOG_PORT    1025
 
-RUN apt update
-
-RUN apt install -y pv sudo bind9-utils python3-pip bash-completion rsync socat oathtool \
-        wget ack automake autoconf nano vim cron mariadb-server
-
-RUN apt install -y python2 && ln -s /usr/bin/python2 /usr/bin/python
+RUN set -eux; \
+    apt-get update -y; \
+    apt-get install -y --no-install-recommends pv sudo bind9-utils python3-pip bash-completion rsync socat oathtool \
+        wget ack automake autoconf nano vim cron mariadb-client python2; \
+    ln -s /usr/bin/python2 /usr/bin/python; \
```

Comment:
Any particular reason to add python2 and make it default here?

Reply:
I don't have much time to test, so I'm just curious.

Reply:
It was just separated to 2 separate RUN commands, I've just merged them together to get smaller image size

Reply:
Apologies, reviewing on mobile, so I missed the context.